### DISCUSSION TITLE
bug  in quickadd settings, capture format

### DISCUSSION THREAD

**Discussion by Littlechay** [chhoumann/quickadd#549] [Q&A]
I am trying to set up a quick add to add tasks to daily notes. 

All goes well until I get to entering a comma after "{{VDATE:Due date"  See screenshot for what I am trying to do and the exact format that I am trying to enter. 
![QuickAdd freezes here](https://github.com/chhoumann/quickadd/assets/21964368/ada1e170-0126-4a4a-8a8c-091ae99cf964)

At that point the format string that I have entered appears below the format entry window and Obsidian freezes completely; the only way out is to kill the app in Taskmanager. See screenshot for what I am trying to do and the exact format that I am trying to enter. 

Two different PCs, Windows 10 on one and W11 on the other. Everything up to date as of now. Installer is v1.3.7

Other plugins installed are:
Calendar
Commander
Dataview
DB Folder
Metadata Menu
Natural Language Dates
Periodic Notes
QuickAdd
Recent Files
Tasks
Templater

Anybody any ideas what I am cocking up ?

**Comment by Cochonnes** [comment]
Hey, I was following the same tutorial and had the same problem. If you go back to the file name on the top, you have to do "Journal/Daily/{{date}}" <- gotta use the curly brackets, that seemed to at least not crash it when putting a comma in the VDATE settings.

**Reply by Littlechay** [reply]
Yes I corrected that after; a bit of research but it didn't solve it for me. 

I have found that if you paste in the capture format it accepts it just fine. So the problem only exists when typing it seems.

**Comment by wardweistra** [comment]
This bug is reported as #455

**Comment by chhoumann** [comment]
Closing the loop on this very late - sorry about that.

You weren't cocking anything up. This was a real bug: typing a comma into a half-finished `{{VDATE:` token made the live format preview lock up Obsidian. @wardweistra had it right - it was tracked as #455 and fixed in release 1.14.0. The preview has since been rewritten so incomplete tokens are simply left as literal text while you type; you can type commas (and anything else) into the format box safely on current versions.

Docs for the VDATE syntax you were setting up: https://quickadd.obsidian.guide/docs/FormatSyntax#vdate

Closing as resolved.
